core/src/main/scala/org/apache/spark/security/CryptoStreamUtils.scala

```diff
@@ -102,4 +150,34 @@ private[spark] object CryptoStreamUtils extends Logging {
     }
     iv
   }
+
+  /**
+   * This class is a workaround for CRYPTO-125, that forces all bytes to be written to the
```

Comment:
There's a pretty nasty workaround for it in the network library... (the non-blocking workaround is a lot worse than this.)